scripts/modules/b2b-account/quotes.js

```diff
@@ -496,10 +498,25 @@ define([
                     sortable: false
                 }
             ],
+            rowActions: [
+                {
+                    displayName: Hypr.getLabel("viewQuoteName"),
+                    action: 'viewQuote'
+                }
+            ],
             relations: {
                 items: Backbone.Collection.extend({
                     model: QuoteModels.Quote
                 })
+            },
+            viewQuote: function (e, row) {
+                e.stopPropagation();
+                var quoteId = row.get('id');
+                if (isSalesRep) {
+                    window.location = '/selleraccount/quote/' + quoteId;
```

Comment:
We don't support sales reps on this page. Only need the buyer link.

Reply:
Actually this should be fine. If we ever enable sales reps for this page it will just work for free.